clang/docs/ReleaseNotes.rst

```diff
@@ -157,6 +157,9 @@ Non-comprehensive list of changes in this release
 - ``__builtin_addc``, ``__builtin_subc``, and the other sizes of those
   builtins are now constexpr and may be used in constant expressions.
 
+- Added ``__builtin_popcountg`` as a type-generic alternative to
+  ``__builtin_popcount{,l,ll}`` with support for any unsigned integer type.
```

Comment:
```suggestion
- ``__builtin_popcountg``, a type-generic alternative to
  ``__builtin_popcount{,l,ll}`` with support for any unsigned integer type, is
  now constexpr and may be used in constant expressions.
```

Reply:
`__builtin_popcountg` already existed; the change is that it’s now constexpr.

Reply:
There was no release note for `__builtin_popcountg` at all yet. See https://github.com/llvm/llvm-project/pull/82359.

Reply:
I see how it is. In that case I’d mention both that it was added and that it’s constexpr.